pandas/conftest.py

```diff
@@ -1598,6 +1598,14 @@ def indexer_sl(request):
     return request.param
 
 
+@pytest.fixture(params=[tm.at, tm.loc])
+def indexer_sa(request):
```

Comment:
maybe `indexer_al` or `indexer_la`?

Reply:
Ok with me, added the s because of the setitem

Reply:
yeah I would ideally make these a bit more descriptive (all of them), at the expense of slightyl increasing length :-> (but let's do this as a followon).

Reply:
Will put up something after this is merged